stock_product_variant_mto/models/product_product.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+IS_MTO_HELP = """
+    Check or Uncheck this field to enable the Make To Order on the variant,
+    independantly from its template configuration.\n
+    Please note that activating or deactivating Make To Order on the template,
+    will reset this setting on its variants.
+"""
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    is_mto = fields.Boolean(
+        string="Variant is MTO",
+        compute="_compute_is_mto",
+        store=True,
+        readonly=False,
+        help=IS_MTO_HELP,
+    )
+
+    route_ids = fields.Many2many(
```

Comment:
If find having this field on `product.product` for handling the feature of this module worrysome